src/features/movie/components/overview/Overview.tsx

```diff
@@ -0,0 +1,26 @@
+import { Movie } from "models/movie.model";
+
+import styled from "./overview.module.scss";
+
+export const Overview: React.FunctionComponent<Movie> = (movie) => {
+  const { description, actors } = movie;
+
+  return (
+    <div className={styled.overview}>
+      <h3>Overview</h3>
+      <div role="document" className={styled.description}>
+        {description}
+      </div>
+      <div className={styled.actors}>
+        {actors.map((actor) => {
+          return (
+            <div className={styled.actor}>
```

Comment:
Needs to have key set to enable React understand list changes - https://legacy.reactjs.org/docs/lists-and-keys.html#keys.

Reply:
Done

Reply:
<div className={styled.actors}>
        {actors.map((actor, index) => {
          return (
            <div className={styled.actor} key={index}>

**like this?**